src/components/Drawer/Drawer.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp } from 'react-icons/io';
+import styles from './Drawer.scss';
+
+const { block, elem } = bem({
+    name: 'Drawer',
+    classnames: styles,
+    propsToMods: ['isOpen'],
+});
+
+const Drawer = props => {
+    const { title, initialIsOpen, children, ...rest } = props;
+    const [isOpen, setIsOpen] = useState(initialIsOpen);
+
+    function handleClick() {
+        setIsOpen(!isOpen);
+    }
+
+    return (
+        <div {...block({ isOpen })} {...rest}>
+            <header {...elem('head', props)}>
+                <h4 {...elem('headTitle', props)}>{title}</h4>
+                <section {...elem('headButtons', props)}>
```

Comment:
`<section>` tag probably is not the best choice here, as far as it assues header and content inside.

![image](https://user-images.githubusercontent.com/403250/64600284-26836080-d3bb-11e9-8798-4f42d3e6cd81.png)

Reply:
Good point!